rpc/src/v1/helpers/errors.rs

```diff
@@ -420,8 +420,8 @@ pub fn transaction_message(error: &TransactionError) -> String {
 	match *error {
 		AlreadyImported => "Transaction with the same hash was already imported.".into(),
 		Old => "Transaction nonce is too low. Try incrementing the nonce.".into(),
-		TooCheapToReplace => {
-			"Transaction gas price is too low. There is another transaction with same nonce in the queue. Try increasing the gas price or incrementing the nonce.".into()
+		TooCheapToReplace { prev, new } => {
+			format!("Transaction gas price {:?}wei is too low. There is another transaction with same nonce in the queue with gas price: {:?}wei. Try increasing the gas price or incrementing the nonce.", new, prev)
```

Comment:
IMO we should print `"{:?}wei"` and "with gas price: {:?}wei"` part only if new and prev are `Some`.